src/components/ui/button/README.md

```
@@ -0,0 +1,119 @@
+      <Link href="/dashboard">Go to Dashboard</Link>
+    </Button>
+  )
+}
+```
+
+## Props
+
+| Prop | Type | Default | Description |
+|------|------|---------|-------------|
+| `variant` | `'solid' \| 'outline' \| 'ghost' \| 'link' \| 'destructive'` | `'solid'` | Visual style variant |
+| `size` | `'sm' \| 'base' \| 'lg' \| 'base'` | Button size |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)